lib/js-api/src/api/content-rest-api/docs/NodesApi.md

```diff
@@ -1196,6 +1198,67 @@ nodesApi.updateNodeContent(`<nodeId>`, contentBodyUpdate, opts).then((data) => {
   console.log('API called successfully. Returned data: ' + data);
 });
 ```
+## initialFolderSizeCalculation
+
+Initiate a new request to calculate folder size.
+
+> this endpoint is available in **Alfresco 7.1** and newer versions.
+
+Initiates a request to calculate the size of the node with identifier **nodeId**.
+
+**Parameters**
+
+| Name                  | Type     | Description                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|-----------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **nodeId**            | string   | The identifier of a node.                                                                                                                                                                                                                                                                                                                                                                                                               |
+
+**Return type**: [JobIdBodyEntry](#JobIdBodyEntry)
+
+**Example**
+
+```javascript
+import {AlfrescoApi, NodesApi} from '@alfresco/js-api';
+
+const alfrescoApi = new AlfrescoApi(/*..*/);
+const nodesApi = new NodesApi(alfrescoApi);
+const contentBodyUpdate = {};
+const opts = {};
+
+nodesApi.initiateFolderSizeCalculation(`<nodeId>`).then((data) => {
+    console.log('API called successfully. Returned data: ' + data);
+});
+```
+## getFolderSizeInfo
+
+Gets the size details of a folder
+
+> this endpoint is available in **Alfresco 7.1** and newer versions.
+
+Fetches the size details of folder with the identifier **nodeId**  
+
+**Parameters**
+
+| Name       | Type     | Description                                                                        |
+|------------|----------|------------------------------------------------------------------------------------|
+| **nodeId** | string   | The identifier of a node.                                                          |
+| **jobId**  | string   | The identifier for the job which is calculating the currently selected node's size  |
```

Comment:
that table is a bit incorrectly formatted (last vertical line | is a bit shifted)

Reply:
Formatted table